### PR TITLE
Add a provider for Keen.io for iOS

### PR DESCRIPTION
**Pull request by cbowns**
Starting a PR for adding Keen.io. This works for me in a local podspec, but a second pair of eyes is A+ helpful.

### CODE REVIEW

File: ARAnalytics.h [hunk 1]
```diff
@@ -80,6 +80,7 @@
 + (void)setupSnowplowWithAddress:(NSString *)address;
 + (void)setupSentryWithID:(NSString *)identifier;
 + (void)setupIntercomWithAppID:(NSString *)identifier apiKey:(NSString *)apiKey;
++ (void)setupKeenWithProjectId:(NSString *)projectId andWriteKey:(NSString *)writeKey andReadKey:(NSString *)readKey;
```
Comment (orta) [inline ARAnalytics.h]: preference here is `ID` over `Id` regardless of what providers think ;)

Comment (cbowns) [inline ARAnalytics.h]: hah! sounds good.

**Comment by orta** [comment]
Everything looks good from the stubbed perspective, this is set up right :+1: 

Would recommend adding a CHANGELOG entry too please

**Comment by cbowns** [comment]
The Keen API is pretty thin, but I think I've got the basics covered. This is ready for review.

**Comment by orta** [comment]
Yep, looks good to me!

**Pull request merged**